src/context/AuthContext.tsx

```diff
@@ -34,7 +34,7 @@ function userCredentialHandler(state, action) {
 const AuthenticationContext = createContext<AuthContextInterface>(null!);
 
 function AuthenticationProvider({ children }: { children: React.ReactNode }) {
-    let [user, setuser] = useState<any>(null);
+    let [user, setuser] = useState<any>(!!localStorage.getItem(VAR_USER_DETAILS) && !!JSON.parse(localStorage.getItem(VAR_USER_DETAILS) ?? ""));
```

Comment:
If variables are not re assigned try using const instead of let

Reply:
done